### PR TITLE
updated dependencies and fixed linter

### PR DESCRIPTION
**Pull request by DarjaLeonova**

### CODE REVIEW

**Review by VladislavsPerkanuks — APPROVED** [review]
Just curious, did you check if the traces still work?

File: go.mod [hunk 1]
```diff
@@ -6,10 +6,10 @@ require (
 	cloud.google.com/go/translate v1.9.3
```
Comment (klavinsjanis) [inline go.mod:6]: does this PR include go version upgrade?

File: Earthfile [hunk 2]
```diff
@@ -3,9 +3,9 @@ PROJECT expect.digital/translate-agent
 
 ARG --global USERARCH # Arch of the user running the build
 
-ARG --global go_version=1.21.3
+ARG --global go_version=1.21.5
 ARG --global golangci_lint_version=1.55.2
```
Comment (klavinsjanis) [inline Earthfile:7]: ARG --global golangci_lint_version=1.55.2 is this the newest?

File: go.mod
Comment (klavinsjanis) [inline go.mod:1]: does this PR include go version upgrade? currently set - go 1.21.3

**Review by janishorsts — APPROVED** [review]

**Pull request merged**